### PR TITLE
Create a nice GitHub profile for Almakers

### PR DESCRIPTION
**Pull request by wo0dyn**
## Rationale

Give our GitHub org a bit of love.

Very simple page to display our stack, values and social links on our GitHub organization[^1].

Thanks to @imleowhite for the Almakers logo. 🙌

## Screenshot

image

👀 You can check the profile online on [Alma test org](https://github.com/alma-test).

## Review

I arbitrarily put the Thierry and tech-leads for review but feel free to assign whoever you want. 😉 

[^1]: https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme

### CODE REVIEW

**Review by rodo — APPROVED** [review]
Thanks for that

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# .github
\ No newline at end of file
+<img src="profile/assets/almakers.png" alt="Almakers" width="20%">
+
+## Alma
+
+At Alma, we put finance at the service of commerce by allowing merchants to increase their sales by 20% on average thanks to payment facilities, without any risk. With Alma, the customer pays later, but the merchant gets paid right away.
```
Comment (Natim) [inline README.md]:
```suggestion
At Alma, we put finance at the service of commerce by allowing merchants to increase their sales by 20% on average. With Alma, the customer pays later, but the merchant gets paid right away with no risk taken.
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# :wave: Hey, we are Almakers!
+
+At Alma, we put finance at the service of commerce by allowing merchants to increase their sales by 20% on average thanks to payment facilities, without any risk. With Alma, the customer pays later, but the merchant gets paid right away.
```
Comment (Natim) [inline profile/README.md]: ditto

**Review by vrialland — APPROVED** [review]
Great idea! Thanks for that! 👏

File: README.md (hunk 1 above)
Comment (wo0dyn) [reply README.md]: Done. Thanks.

**Review by Natim — APPROVED** [review]

**Review by gscout — APPROVED** [review]

**Pull request merged**